src/events.ts

```diff
@@ -213,6 +213,9 @@ export type ConnectedAccountCompletedFirstSync =
 export type ConnectedAccountDeleted =
   CommonConnectedAccountEvent<"connected_account.deleted">
 
+export type ConnectedAccountCompletedSyncAfterReconnection =
+  CommonConnectedAccountEvent<"connected_account.completed_sync_after_reconnection">
```

Comment:
I know this is already pretty long but my first thought was maybe this should be `completed_first_sync_after_reconnection` to convey its similarity to the existing one for initial connection